colibre/auto_plotter/gas_fraction_plots.yml

```diff
@@ -0,0 +1,183 @@
+h2_frac_func_stellar_mass:
+  type: "scatter"
+  legend_loc: "lower left"
+  selection_mask: "derived_quantities.xcoldgass_sel"
+  comment: r"$10^9 M_{\odot} < M_{\ast} < 10^{11.5} M_{\odot}$"
+  x:
+    quantity: "apertures.mass_star_30_kpc"
+    units: Solar_Mass
+    start: 1e9
+    end: 4e11
+  y:
+    quantity: "derived_quantities.h2_to_stellar_mass"
+    units: "Solar_Mass / Solar_Mass"
+    start: 1e-4
+    end: 100
+  median:
+    plot: true
+    log: true
+    number_of_bins: 9
+    start:
+      value: 1e9
+      units: Solar_Mass
+    end:
+      value: 4e11
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-H2 Fraction
+    section: H_2 Fractions
+  observational_data:
+    - filename: GalaxyH2Fractions/Saintonge2017_abcissa_M_star.hdf5
+
+h2_frac_func_ssfr:
+  type: "scatter"
+  legend_loc: "upper right"
+  selection_mask: "derived_quantities.xcoldgass_sel"
+  comment: r"$10^9 M_{\odot} < M_{\ast} < 10^{11.5} M_{\odot}$"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_30_kpc"
+    units: "1 / Gigayear"
+    start: 1e-4
+    end: 1e4
+  y:
+    quantity: "derived_quantities.h2_to_stellar_mass"
+    units: "Solar_Mass / Solar_Mass"
+    start: 1e-4
+    end: 100
+  median:
+    plot: true
+    log: true
+    number_of_bins: 29
```

Comment:
This should be 40 to be consistent with the rest using 0.2 dex bins. Also worth using `adaptive: true` everywhere.